CONTRIBUTING.md

```diff
@@ -208,6 +208,29 @@ Work has been proposed to add a separate installer for AUR packages [ros-infrast
 
 For pip installers they are expected to be in the main PyPI index https://pypi.org/.
 
+### pkg-config rules
```

Comment:
@christianrauch hmm, what do you mean by `pkg-config rules`? This is just a guideline for `rosdep` rule key naming.

Reply:
The idea of this PR is that if there is a pkg-config file, the rosdep key should be derived from this, so that dependency names are more standardised across distributions and build systems, and that this could be automated in future.